src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -43,8 +48,24 @@ case class WorkflowDescriptor(id: WorkflowId,
   val name = namespace.workflow.unqualifiedName
   val actualInputs: WorkflowCoercedInputs = coercedInputs ++ declarations
   val props = sys.props
+  val relativeWorkflowRootPath = s"$name/$id"
+  val wfOutputsRoot = workflowOptions.get("outputs_path").toOption
   lazy val fileHasher: FileHasher = { wdlFile: WdlFile => SymbolHash(ioManager.hash(wdlFile.value)) }
 
+  // GCS FS with the workflow working directory as root
+  val gcsFilesystem = for {
+    interface <- gcsInterface.toOption
```

Comment:
Outside of the point which was raised earlier as to whether or not losing the exception is correct here I'm curious why you're doing it this way on these, couldn't you have the option and then map it?